roles/common/handlers/main.yml

```diff
@@ -0,0 +1,13 @@
+- name: stop monit service
```

Comment:
This is confusing. It's called "stop monit service", but stops delayed job.

Reply:
Nice catch, that was a mistake.